### DISCUSSION TITLE
jsPsych with React

### DISCUSSION THREAD

**Discussion by ZahinPrangon** [jspsych/jsPsych#620] [Q&A] [answered]
How to integrate jsPsych with React? I download the npm package how to declare jsPsych when initializing its instance?

**Comment by jodeleeuw** [comment]
The npm package isn't officially part of jspsych yet. It's generated by a third party.

Once we convert jspsych to be CommonJS module friendly, we'll get an official package released.

@marielajennings do you have any insight on the current npm package?

**Comment by ZahinPrangon** [comment]
Thanks! Is there any way I can contribute in making that happen or atleast learn how to do it?

**Comment by jodeleeuw** [comment, marked as answer]
We're tracking work on this in #529. Happy to have help!

**Comment by jdpigeon** [comment]
We're working on integrating jspsych into react [here](https://github.com/makebrainwaves/jspsych-react)
